ci-operator/step-registry/fips-check/fips-or-die/fips-check-fips-or-die-commands.sh

```diff
@@ -121,14 +121,16 @@ if [[ $ret -eq 0 ]]; then
     fi
 fi
 
-node_name=`oc get node -l node-role.kubernetes.io/master= -o=jsonpath="{.items[0].metadata.name}"`
-arch=`oc get node/$node_name -o=jsonpath="{.status.nodeInfo.architecture}"`
-if [[ $arch != "amd64" ]]; then
-    echo "Skip this test since it doesn't support $arch."
-    exit 0
-else
-    echo "Start to run test on $arch"
-fi
+mapfile -t node_archs< <(oc get nodes -o jsonpath --template '{range .items[*]}{.status.nodeInfo.architecture}{"\n"}{end}' | sort -R | uniq -d)
+echo "The architectures included in the current cluster nodes are ${node_archs[*]}"
+### Skip ppc64le and s390x temporarily becasue the test images are not manifest list images now.
+NON_SUPPORTED_ARCHES=(arm64 ppc64le s390x)
+for arch in "${node_archs[@]}"; do
+  if [[ "${NON_SUPPORTED_ARCHES[*]}" =~ $arch ]]; then
+      echo "Skip this test since it doesn't support $arch."
+      exit 0
+  fi
+done
```

Comment:
Hi ... Is there an issue tracking when the images will be manifest listed?

Reply:
waiting for @aleskandro back from vacation, we will have a discussion and task creation.

Reply:
great - thank you

Reply:
@prb112 FYI. here is the jira issue https://issues.redhat.com/browse/MIXEDARCH-307